### ISSUE TITLE
代码与文章描述存在偏差

### ISSUE THREAD

**Issue by WWWWWWWP** [anyuan1999/STGAN#1]
感谢您公开这项工作
在文章的描述中，您使用XGBoost作为异常检测的核心模块。然而在存储库中，异常检测使用了Threatrace中描述的方式进行。

此外，我在当前配置下对trace数据进行实验，但结果几乎检测不到任何恶意行为，请问是否存在遗漏的步骤导致实验结果的偏差。

**Comment by anyuan1999** [comment]
Hello, wwwwwwwp! Thank you very much for your interest in STGAN. I apologize for the delayed response. Upon verification, we found that the current version of the code has some minor issues when training on the trace dataset, specifically, the learning rate needs to be adjusted to achieve optimal performance (Currently, the lr in train_trace.py is 0.1). We will soon upload a set of pre-trained model weights along with the detection code that includes the XGBoost-based classifier for quick verification.

**Comment by yofierl** [comment]
你好，Dr. Shang，感谢您公开这项工作，我目前在trace与cadets数据集上已经成功复现，但在theia数据集上问题比较大，并且关于恶意节点，是把“从来没被任何一次模型正确预测过”的节点当作最终的可疑／恶意预测结果，这样理解对吗？然后，关于嵌入维度，最佳的嵌入维度是128，那相应的GAT输出也应该是128，TGN则为256，那么TGN的输出通道32是不是有点过于小了，最后期待您包括基于 XGBoost 的分类器的检测代码。

**Comment by anyuan1999** [comment]
Hello, yofierl! Sorry for delayed response. I've been quite busy recently. The code is expected to be uploaded within two weeks. Please wait. ：)

**Comment by yofierl** [comment]
Eagerly awaiting your upload.

**Comment by anyuan1999** [comment]
> Eagerly awaiting your upload.

It has been uploaded. Please pull and test.
